incubator/istio/templates/rbac.yaml

```diff
@@ -1,10 +1,17 @@
+{{- if and .Values.istio.install (not .Release.IsInstall) -}}
```

Comment:
One file per resource, please.

Reply:
ouch! Also for the CRD and customresources, potentially hundreds of files... Can we get consensus on what we should do here?

Reply:
@viglesiasce any opinions on this?

Reply:
I'm fine with multiple resources per file if it makes your life as a chart maintainer easier. As a maintainer of the repo it doesn't make things harder or easier for me.

Reply:
Yes, +1 for this, I think that we can treat the RBAC as an exception for this, maintaining in one file is more easy to understand.

Reply:
OK, then let's keep it like this. I think, in general we recommend one file per resource, but I guess an exception makes sense here.